pkg/goDB/conditions/node/parse.go

```diff
@@ -318,7 +318,7 @@ func (p *parser) condition() (result Node) {
 // Corresponds to grammar rule "attribute"
 func (p *parser) attribute() (result string) {
 	attributes := []string{
-		types.DIPName, types.SIPName, "dnet", "snet", types.DportName, types.ProtoName, // non-sugar
+		types.DIPName, types.SIPName, "dnet", "snet", types.DportName, types.ProtoName, FilterKeywordDirection, // non-sugar
```

Comment:
- [ ] Below: add "sugar" for the direction: `"dir"`
- [ ] Move `FilterKeywordDirection` to `types.FilterKeywordDirection` to be consistent with the rest of the line